### PR TITLE
Navbar, navbar buttons and logo refactor

### PR DESCRIPTION
**Pull request by FireQueen-3010**
image
image
image

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
Do review @ArunTeltia :)

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.js [hunk 1]
```diff
@@ -131,17 +132,23 @@ export default function PersistentDrawerLeft() {
...
               />
             </IconButton>
             {!open && (
               <div
-                style={{ display: "flex", alignItems: "center", gap: "2px" }}
+                
               >
                 <div>
-                  <h4>Project Sakura</h4>
+                  <h2 className={"Main-logo"}><span>Project</span>
```
Comment (ArunTeltia) [inline src/components/PersistentDrawerLeft/PersistentDrawerLeft.js:145]: you can remove the bracket  from the className

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.js [hunk 2]
```diff
@@ -131,17 +132,23 @@ export default function PersistentDrawerLeft() {
...
+                  <span className={"Sakura"}> Sakura</span></h2>
```
Comment (ArunTeltia) [inline src/components/PersistentDrawerLeft/PersistentDrawerLeft.js:146]: here as well

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.js [hunk 3]
```diff
@@ -131,17 +132,23 @@ export default function PersistentDrawerLeft() {
...
+                </div>
+                <div className={"spring"}>
```
Comment (ArunTeltia) [inline src/components/PersistentDrawerLeft/PersistentDrawerLeft.js:148]: here as well

File: src/components/BreadCrums/BreadCrum.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ const BreadCrum = () => {
   return (
     <div className="breadcrum">
       <div className="breadcrum_content">
-        <HomeIcon style={{ backgroundColor: "#1e1e1e" }} /> /{" "}
+        <HomeIcon className="breadcrum-home" /> /{" "}
```
Comment (ArunTeltia) [inline src/components/BreadCrums/BreadCrum.js:17]: can we follow the underscore css as it is easier to handle instead of -

Comment (ArunTeltia) [reply src/components/BreadCrums/BreadCrum.js:17]: great catch btw

File: src/index.css [hunk 5]
```diff
@@ -3,9 +3,11 @@
...
+  /* background-color: black; */
 }
 a {
   text-decoration: none;
-  color: black;
-  
 }
+
+
+
```
Comment (ArunTeltia) [inline src/index.css:13]: can we remove these 3 lines as well these are redundant

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

File: src/components/BreadCrums/BreadCrum.js (hunk 4 above)
Comment (FireQueen-3010) [reply src/components/BreadCrums/BreadCrum.js:17]: I didn't understand this

**Comment by FireQueen-3010** [comment]
Done @ArunTeltia

**Review by ArunTeltia — APPROVED** [review]

**Pull request merged**